src/serac/physics/utilities/variational_form/tests/weak_form_comparisons.cpp

```diff
@@ -237,9 +260,9 @@ void weak_form_test(mfem::ParMesh& mesh, Hcurl<p> test, Hcurl<p> trial, Dimensio
       Dimension<dim>{},
       [&](auto x, auto vector_potential) {
         auto [A, curl_A] = vector_potential;
-        auto J = a * A - tensor<double, dim>{10 * x[0] * x[1], -5 * (x[0] - x[1]) * x[1]};
-        auto H = b * curl_A;
-        return std::tuple{J, H};
+        auto J_term      = a * A - tensor<double, dim>{10 * x[0] * x[1], -5 * (x[0] - x[1]) * x[1]};
+        auto H_term      = b * curl_A;
+        return std::tuple{J_term, H_term};
```

Comment:
what is motivating these name changes?

Reply:
Since `J` was declared in the containing scope as well, this was issuing a shadowing warning.